<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectPathWidget</class>
 <widget class="QWidget" name="SelectPathWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="DiscriptionLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="FilePathBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="SelectDialogBtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EraseBtn">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
